containers/base/Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
 
 # Setup Node.js
```

Comment:
Well we're only choosing to pin to a version that works, do we have any other reason? It's just like any other installation in the Dockerfile.